src/tui/mod.rs

```diff
@@ -50,7 +50,7 @@ pub enum Window {
     Home,
     Workspaces { index: usize, empty: bool },
     Tags { index: usize, empty: bool },
-    WindowRules { index: usize, empty: bool },
```

Comment:
I would prefer this stays as `WindowRules` instead `Rules`, because even though it is `Window::WindowRules`, both windows in that are completely disconnected, the window enum is referring to windows in the tui, while the `WindowRules` refer to LeftWM window rules.